website/content/docs/widgets/markdown.md

```diff
@@ -22,3 +22,9 @@ _Please note:_ in case you want to use your markdown editor to fill a markdown's
 This would render as:
 
 ![Markdown widget example](/img/widgets-markdown.png)
+
+_Please note:_ If you are building a site with [Hugo](https://gohugo.io), you will need to add the [`markdownify`](https://gohugo.io/functions/markdownify/) filter function when displaying your output in order for the markdown to be parsed. For example:
```

Comment:
@verythorough Any thoughts on this? I'm still not so sure it's a good idea to mention Hugo here, when we could make a more general reminder instead.

Reply:
@edadams If we can make this less Hugo-specific, that would be great!

Reply:
As requested, I've rewritten my changes to not mention Hugo. I feel that with the documentation now being very vague, it is significantly less helpful.

When I spoke to @tech4him1 about this problem in issue #1596, I got the impression that this was a problem only Hugo users would experience as Hugo is the only SSG that does this. That is why my changes mentioned Hugo.

The new change is as follows:

>_Please note:_ If you are building a site with certain static site generators, you may need to add a markdownify filter when you come to display the output. Consult with your static site generator's documentation for rendering markdown.

I have intentionally removed the link to [Hugo's `markdownify` filter](https://gohugo.io/functions/markdownify/) as I believe that is what was requested, but as mentioned above I think it significantly less helpful.

I feel quite convinced that a user in my position, trying to get started with Netlify and Hugo, wouldn't know why their Markdown files weren't being parsed. In my head, my first thought was to see if Netlify was the problem. I suppose that someone else could assume it was a Hugo problem. It isn't intuitive that Hugo requires a filter, but that isn't your fault. As far as Netlify is concerned, it's doing its job right. 

What I'm saying is I see the reason to want it vague but I worry it becomes less helpful as a result of that. Thoughts? :)